contracts/test/tx_allow_list.ts

```diff
@@ -68,4 +68,12 @@ describe("ExampleTxAllowList", function () {
   test("should not let allowed to revoke admin", "step_onlyAdminCanRevoke")
 
   test("should let admin to revoke allowed", "step_adminCanRevoke")
+
+  test("should let manager to revoke allowed", "step_managerCanRevoke")
+
+  test("should let manager to add allowed", "step_managerCanAllow")
+
+  test("should not let manager to add admin", "step_managerCannotGrantAdmin")
+
+  test("should not let manager to revoke admin", "step_managerCannotRevokeAdmin")
 })
```

Comment:
should we add " should let manager deploy"

Reply:
also manager cannot add another manager

Reply:
added